libraries/Microsoft.Bot.Schema/Activity.cs

```diff
@@ -1,14 +1,4 @@
-// <auto-generated>
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// </auto-generated>
-
```

Comment:
I totally get why you want to go this way, but I think this is a non-starter.  We are constantly updating the schema for new types, and have to keep the changes in sync with multiple languages. 
I just don't see that we have the bandwidth to make this 100% hand coded.  This is a huge delta, and I wish you had talked with me before embarking on it.